### PR TITLE
Fix typos in User Guide

### PR DESCRIPTION
**Pull request by w2vgd**
Update section numberings and update outdated examples

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -50,40 +51,42 @@ This user guide uses various formatting styles to facilitate reading and to comm
...
-1. Now let's try adding a new appointment. <br>Type `add appointment n/Meet Jacob for dinner r/At Orchard Central's food court d/19-4-2021 t/1930` and hit the <kbd>Enter</kbd> button. Scroll down to the bottom of the appointment list to see your newly added appointment.
-1. Forgot when you are supposed to meet Simon again? Let's try finding out! Type `find appointment simon` and press <kbd>Enter</kbd>. There it is! Your appointment with Simon is on Sep 20, 2021.
+1. Let's try out your first command by adding a new property! Type <br>`add property n/Bishan t/Hdb a/Blk 150 Bishan Street 11 #02-101 p/570150 d/30-6-2021` into the command box and press <kbd>Enter</kbd> to execute it. After that, scroll down to the bottom of the property list to see your newly added property.
+1. Want to view the properties in order of their asking price? Type the command <br>`sort property o/desc k/price` and press <kbd>Enter</kbd>. The property list will now be ordered in descending order, with the property having the highest asking price at the top.
```
Comment (candyhy) [inline docs/UserGuide.md]: How about 'the property list will now be in descending order based on price'?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -136,19 +139,17 @@ Format: `add appointment n/NAME r/REMARKS d/DATE t/TIME​`
...
-### Editing
-
-#### Editing a property : `edit property`
+### 3.3 Editing
 
-Overwrites the information of the property according to the flags provided.
+#### 3.3.1 Editing a property : `edit property`
 
-Formats: `edit property INDEX [n/NAME] [t/PROPERTY_TYPE] [a/ADDRESS] [p/POSTAL_CODE] [d/DEADLINE] [r/REMARKS] [cn/CLIENT_NAME] [cc/CLIENT_CONTACT_NUMBER] [ce/CLIENT_EMAIL] [ca/CLIENT_ASKING_PRICE] [tags/TAGS_SEPARATED_BY_COMMAS]​`
+Overwrites the information of the property according to the parameters provided.
```
Comment (candyhy) [inline docs/UserGuide.md]: I think this doesn't sound very user friendly? A user may not know what a parameter is

File: docs/UserGuide.md [hunk 3]
```diff
@@ -192,24 +193,24 @@ Examples:
...
 Shows a list of all appointments in the app.
 
 
-### Updating status
+### 3.6 Updating status
 
-#### Updating the status of a property : `update`
+#### 3.6.1 Updating the status of a property : `update`
 
 Updates the status of a property from Option to Purchase, to Sales and Purchase Agreement to Completion
```
Comment (candyhy) [inline docs/UserGuide.md:215]: should we explain what does Option, Sales and Purchase Agreement, and Completion mean?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -229,15 +230,15 @@ Examples:
 *  `update 1 u/new 600000` Creates a new status with amount 600000 for the 1st property.
 *  `update 3 u/proceed` Moves the status of the 3rd property to next one.
```
Comment (candyhy) [inline docs/UserGuide.md:231]: Don't think the user will know what does status refer to

File: docs/UserGuide.md [hunk 5]
```diff
@@ -247,20 +248,20 @@ Examples:
...
+* `find property [KEYWORD]... [OPTION]...`
 
 Description:
 * There can be 0 or more keywords and 0 or more options, but keywords and options cannot be both empty. All text are case insensitive. 
 
 Options:
-* `t/[PROPERTY_TYPE]`
+* `[t/PROPERTY_TYPE]`
 
     Search for properties whose housing type field contain patterns specified in `[REMARKS]`.
```
Comment (candyhy) [inline docs/UserGuide.md]: is this supposed to be `[PROPERTY_TYPE]` instead?

File: docs/UserGuide.md [hunk 6]
```diff
@@ -296,45 +297,50 @@ Examples:
 * `find appointment bob`
 * `find appointment sunday`
 
-#### Searching clients: `find client`
+#### 3.8.3 Searching clients: `find client`
 
 Finds appointments that matches the keywords and properties whose clients matches the same keywords. Both are done at the same time.
```
Comment (candyhy) [inline docs/UserGuide.md:302]: think the 'Both done at the same time' is unnecessary? If anything, maybe 'simultaneously' will be better

**Review by candyhy — COMMENTED** [review]
LGTM with some comments

File: docs/UserGuide.md (hunk 1 above)
Comment (w2vgd) [reply docs/UserGuide.md]: ohh ok!

File: docs/UserGuide.md (hunk 2 above)
Comment (w2vgd) [reply docs/UserGuide.md]: Will update this

File: docs/UserGuide.md (hunk 3 above)
Comment (w2vgd) [reply docs/UserGuide.md:215]: @dvdweien to update this section

File: docs/UserGuide.md (hunk 4 above)
Comment (w2vgd) [reply docs/UserGuide.md:231]: @dvdweien to update this section too

File: docs/UserGuide.md (hunk 5 above)
Comment (w2vgd) [reply docs/UserGuide.md]: The format is supposed to be `t/hdb` for example

File: docs/UserGuide.md (hunk 6 above)
Comment (w2vgd) [reply docs/UserGuide.md:302]: @w-yuchen to update this section

File: docs/UserGuide.md (hunk 5 above)
Comment (candyhy) [reply docs/UserGuide.md]: I meant line 265 266, there's this `[remarks]`, this that supposed to be the case?

File: docs/UserGuide.md [hunk 7]
```diff
@@ -54,7 +54,7 @@ This user guide uses various formatting styles to facilitate reading and to comm
 ### 2.2 Trying out
 
 1. Let's try out your first command by adding a new property! Type <br>`add property n/Bishan t/Hdb a/Blk 150 Bishan Street 11 #02-101 p/570150 d/30-6-2021` into the command box and press <kbd>Enter</kbd> to execute it. After that, scroll down to the bottom of the property list to see your newly added property.
-1. Want to view the properties in order of their asking price? Type the command <br>`sort property o/desc k/price` and press <kbd>Enter</kbd>. The property list will now be ordered in descending order, with the property having the highest asking price at the top.
+1. Want to view the properties in order of their asking price? Type the command <br>`sort property o/desc k/price` and press <kbd>Enter</kbd>. The property list will now be in descending order based on price.
```
Comment (candyhy) [inline docs/UserGuide.md]: I think can keep the part act the first property having highest price at the book though, although the phrasing for the first half is good now

File: docs/UserGuide.md (hunk 5 above)
Comment (w2vgd) [reply docs/UserGuide.md]: ohh, thanks for catching this

File: docs/UserGuide.md (hunk 7 above)
Comment (w2vgd) [reply docs/UserGuide.md]: Shall update this

**Review by candyhy — APPROVED** [review]
LGTM

**Pull request merged**